vendor/assets/stylesheets/rolodex/settings/utilities/_layout.sass

```diff
@@ -0,0 +1,51 @@
+// Layout
+// ========================================
+
+// Floats
+
+.pull-left
+  float: left
+
+.pull-right
+  float: right
+
+// Clearfix
+
+.group
+  +group
+
+// Grid
+// ========================================
+
+.grid
+  list-style: none
+  +rem(letter-spacing, -4px)
+
+.opera:-o-prefocus,
```

Comment:
Neither.

Browsers that don't understand `:-o-prefocus` will completley disregard this ruleset, and skip over it. Opera, will read this ruleset though but since `.opera` isn't a used class this selector won't do anything but the `.grid` class below it will do the trick we need it to.